.github/pull_request_template.md

```diff
@@ -0,0 +1,63 @@
+<!--
+Thanks for submitting a pull request! Here are some helpful tips:
+
+* Always create branches on and target the `develop` branch.
+* Run all the tests locally and ensure that they are passing.
+* Run `cargo fmt --all` to ensure that the code is formatted.
+* Small commits and contributions which attempt one single goal is preferable.
+* If the idea changes or adds anything functional which will affect users an AIP
+discussion is required first on the Aurora forum: 
+https://forum.aurora.dev/discussions/AIPs%20(Aurora%20Improvement%20Proposals).
+* Avoid breaking the public API (namely in engine/src/lib.rs) unless required.
+* If your PR is a WIP, ensure that you enable "draft" mode to ensure that it is
+not reviewed too early.
+* Your first PRs won't use the CI automatically unless started by a maintainer.
+If this is not your first PR, please do NOT abuse the CI resources.
+
+Checklist:
+- [ ] I have performed a self-review of my code
+- [ ] I have documented my code, particularly in the hard to understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] I have added tests to prove my fix or new feature is effective and works
+- [ ] Any dependent changes have been merged
+- [ ] The PR is targeting the `develop` branch and not `master`
+- [ ] I have pre-squashed my commits into a single commit and rebased.
+-->
+
+## Description
+
+<!-- 
+Provide a general summary of your changes. A clear overview along with an 
+in-depth explanation is very helpful.
+
+If this PR closes any issues be sure to add "closes PR #<number>" somewhere.
+-->
+
+## Performance / NEAR gas cost considerations
+
+<!--
+Performance regressions are not ideal. Any PR must be completely mindful of any
+gas cost increases. The CI will fail if the gas costs change at all. It is good
+to provide an explanation of this change, if necessary.
+-->
```

Comment:
Maybe mention that failing tests due to _lower_ gas costs are a good thing. It might not be obvious for a newcomer and they could worry about breaking tests.